src/oc_erchef/elvis.config

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
Let's add a comment to reference the config docs, https://github.com/inaka/elvis_core/wiki/Rules